packages/Stylers/src/helpers.js

```diff
@@ -1,12 +1,19 @@
 import tokens from "@paprika/tokens";
 
+//
+// Utilities
+//
+
+export const toInt = token => Number.parseInt(token, 10);
+
 //
 // Typography
 //
 
 const fontScaleRatio = 1.125; // 8:9 - Major second
 
-const fontSizeValue = (scale = 0) => Math.round(Number.parseInt(tokens.fontSize.default, 10) * fontScaleRatio ** scale);
+export const fontSizeValue = (scale = 0) =>
+  Math.round(Number.parseInt(tokens.fontSize.default, 10) * fontScaleRatio ** scale);
```

Comment:
I also dont like this and wish it were just a map. Not relevant to this PR, though.

https://www.amazon.ca/Dont-Make-Think-Revisited-Usability/dp/0321965515/ref=pd_sbs_14_img_0/144-3293126-0814043?_encoding=UTF8&pd_rd_i=0321965515&pd_rd_r=005d21f3-1cf3-41ce-bef8-f44eb4b18598&pd_rd_w=oxoLl&pd_rd_wg=5HCng&pf_rd_p=9926bb69-42b9-46e4-b788-f665992e326d&pf_rd_r=C4WXAX75XRKMZV2PY9T3&psc=1&refRID=C4WXAX75XRKMZV2PY9T3

Reply:
Agreed, we can revisit in future but i'm a fan of simplification as well!